site/en/guide/keras/train_and_evaluate.ipynb

```diff
@@ -375,8 +375,8 @@
       },
       "outputs": [],
       "source": [
-        "model.compile(optimizer=keras.optimizers.RMSprop(learning_rate=1e-3),\n",
-        "              loss=keras.losses.SparseCategoricalCrossentropy(from_logits=True),\n",
```

Comment:
@MarkDaoust I think `from_logits=True` is the preferred usage these days, yes?

Reply:
Exactly.

I reverted that line, and updated the text.